website/docs/integrator-section/verifiable-document/ethereum/oa-embedded-html.md

```diff
@@ -0,0 +1,100 @@
+
+## Create your Verify URL
+
+To get started, we will need to have an issued/signed OA document.
+
+- In this guide, we will be using our [sample HealthCert document](https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/master/test/fixtures/v2/pdt_pcr_notarized_with_nric_wrapped.json).
+
+If you have your own OA document, you can use it as well.
+
+### Encoding URL
+
+1. Copy the [sample document contents](https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/master/test/fixtures/v2/pdt_pcr_notarized_with_nric_wrapped.json). If you are using your own OA document, open the file in a text editor and copy the file data.
+
+   ![Copy document content](/docs/integrator-section/verifiable-document/ethereum/oa-embedded-html/copy-healthcert-data.png)
+
+2. Minify/ Compress the OA document data
+
+   - We will be minifying/compressing the JSON data in the OA document to prevent additional characters from being encoded later in step (3)
+   - You can use online tools like [Code Beautify](https://codebeautify.org/jsonminifier) to help you
+
+     ![Paste document for encoding](/docs/integrator-section/verifiable-document/ethereum/oa-embedded-html/minify-document-data.png)
+
+   - Copy and paste the data, into a text editor (we will need it later)
+
+3. Encode your data using a URL Encoder
+
+   - We will be encoding the data from step (2) so that it can be appended to the back of the Verify URL later on
+   - You can use online tools like [urlencoder.org](https://www.urlencoder.org/) to help you
+     ![Paste document for encoding](/docs/integrator-section/verifiable-document/ethereum/oa-embedded-html/encode-document-data.png)
+   - Copy and paste the encoded data into a text editor (we will need it later)
+
+### Create the Verify URL with OA Document Data
+
+- Open a text editor of your choice
+- Add the encoded data, which you have copied from earlier, after the hash `#` symbol in `https://www.verify.gov.sg/verify?m=uri-fragment#`.
+
+```bash
+https://www.verify.gov.sg/verify?m=uri-fragment#<paste_your_encoded_data_here>
+```
+
+## Creating The HTML File
+
+### Sample HTML File
+
+To standardise the HTML files being issued, HealthCerts Providers are encouraged to use the [sample HTML file](https://github.com/Open-Attestation/oa-embedded-html/blob/master/samples/healthcert-pdt-oa-embedded-uri-fragment-sample.html).
+
+For simplicity, we will be using the contents from the [sample HTML file](https://github.com/Open-Attestation/oa-embedded-html/blob/master/samples/healthcert-pdt-oa-embedded-uri-fragment-sample.html). Then we will embed the Verify URL into the "Proceed" button so that users will be redirected to the Verify page upon clicking.
+
+### Embed URL into Action Button
```

Comment:
```suggestion
### Replace the button link with Verify URL
```

Reply:
Is it better to write "Replace the button link with your Verify URL" to indicate that the user has to use their newly created Verify URL?